peachjam/views/accounts.py

```diff
@@ -0,0 +1,26 @@
+from django.contrib.auth import get_user_model
+from django.urls import reverse
+from django.views.generic import UpdateView
+
+from peachjam.forms import UserEditorForm
+
+User = get_user_model()
+
+
+class EditAccountView(UpdateView):
+    authentication_required = True
+    model = User
+    template_name = "user_account/edit.html"
+    form_class = UserEditorForm
+
+    def get_success_url(self):
+        return reverse("edit_account")
+
+    def get_object(self, queryset=None):
+        return self.request.user
+
+    def get_initial(self):
+        initial = super(EditAccountView, self).get_initial()
```

Comment:
I don't think this method is necessary because the form will be initialised from the properties on `get_object`